cypress.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = defineConfig({
       // Default to sample flagged record set on full
       FLAGGED_RECORD_SET_ID: process.env.E2E_FLAGGED_RECORD_SET_ID || 'a0r0P00002WqGZ6QAN',
       // Default to 400 China Basin in full
-      FCFS_LISTING_ID: process.env.FCFS_LISTING_ID || 'a0W4U00000Nlb5lUAB',
+      FCFS_LISTING_ID: process.env.FCFS_LISTING_ID || 'a0W6s000008CO5yEAG',
```

Comment:
suggestion: rename this constant to include rental in the name so it is clear the kind of fcfs listing

Reply:
love it!